package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "jigmi",
+  "version": "0.0.0",
+  "description": "Jitsi Generic Monitoring Infrastructure",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/jitsi/jigmi"
+  },
+  "keywords": [],
+  "author": "bbaldino",
+  "readmeFilename": "README.md",
+  "dependencies": {
+    "babel-core": "^6.26.0",
+    "babel-loader": "^7.1.2",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-react": "^6.24.1",
+    "body-parser": "^1.18.2",
+    "express": "^4.16.2",
+    "highcharts": "^6.0.2",
+    "prop-types": "^15.6.0",
+    "react": "^16.0.0",
+    "react-dom": "^16.0.0",
+    "react-highcharts": "^13.0.0",
+    "react-redux": "^5.0.6",
+    "redux": "^3.7.2",
+    "sqlite3": "^3.1.13",
+    "webpack": "^3.8.1"
+  },
+  "devDependencies": {
+  },
+  "scripts": {
+    "postinstall": "webpack",
+    "validate": "npm ls",
+    "start": "node server.js"
+  },
+  "pre-commit": [
+  ],
+  "main": "./index.js",
+  "license": "Apache-2.0"
+}
```

Comment:
Adding the node version here might be good practice. It's a lot like specifying the package version explicitly instead of doing a *.

Reply:
you mean like:
```
   "engines": {
     "node": ">=6.0.0"
   },
```
? i'll add that